qiskit_nature/operators/second_quantization/qubit_converter.py

```diff
@@ -24,13 +26,15 @@
 from qiskit_nature import QiskitNatureError
 from qiskit_nature.mappers.second_quantization import QubitMapper
 
-from . import SecondQuantizedOp
+from qiskit_nature.operators.second_quantization import SecondQuantizedOp
 
 logger = logging.getLogger(__name__)
 
 
 class QubitConverter:
-    """A converter from Second-Quantized to Qubit Operators.
+    """ **DEPRECATED** A converter from Second-Quantized to Qubit Operators.
+
+    The QubitConventer was moved to `converters.second_quantization`
```

Comment:
Do we need to say `qiskit_nature.converter.second_quantization`?

Reply:
This has never been included in the documentation so any text there would just be read when you view the source. I was hoping that since we only just released not too many people yet would be using it from where it was. We did in test cases and notebooks, but I changed all that to the new location. This was kept just in case any body started using it.